src/20_Components/amp-form.html

```diff
@@ -205,6 +205,46 @@
     </div>
   </form>
 
+  <!-- ## Form Verification -->
+  <!--
+  Form verification enables asynchronous validation of form fields. As a user
+  fills out a form with verification enabled, `amp-form` sends the
+  values to the `verify-url` to check the value using logic on the server.
+  If an error occurs, such as a username being taken, it will show as a
+  validation error in the form.
+
+  Read [here](https://www.ampproject.org/docs/reference/components/amp-form)
+  for complete documentation.
+
+  Use the `verify-xhr` attribute to enable the form verification feature.
+
+  Form verification can combine with custom error reporting using
+  `visible-when-invalid="customError"`.
```

Comment:
`customError` is part of the [HTML5 Constraint Validation spec](https://dev.w3.org/html5/spec-preview/constraints.html#the-constraint-validation-api), like `patternMismatch` or  `valueMissing`. AMP uses `setCustomValidity` for the verification errors, which applies the `customError` validity to an element.